clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -157,14 +197,26 @@ object UncommittedGarbageCollector {
       firstSlice: String,
       startTime: java.time.Instant,
       success: Boolean,
+      expiredAddresses: DataFrame,
```

Comment:
I think this is redundant - this is in fact the same DF as removed, there's no difference except the naming regardless of the GC operation we are doing.